src/main/java/net/openhft/chronicle/core/io/Closeable.java

```diff
@@ -46,7 +46,7 @@ static void closeQuietly(@Nullable Object o) {
             try {
                 ((java.lang.AutoCloseable) o).close();
             } catch (Exception e) {
-                Jvm.debug().on(Closeable.class, e);
+                Jvm.warn().on(Closeable.class, e);
             } catch (Throwable e) {
```

Comment:
You could remove the catch exception block altogether because it's the same as the catch throwable one below it now?

Reply:
yes for sure. Maybe we consider adding a message:
```
            try {
                ((java.lang.AutoCloseable) o).close();
            } catch (Throwable e) {
                Jvm.warn().on(Closeable.class, "Problem calling close(). Ignoring", e);
            }
```